ragstack-e2e-tests/pyproject.langchain.toml

```diff
@@ -3,7 +3,7 @@ name = "ragstack-e2e-tests"
 version = "0.1.0"
 description = "RAGStack tests"
 license = ""
-authors = ["DataStax"]
```

Comment:
These are the default settings of the [even-better-toml](https://marketplace.visualstudio.com/items?itemName=tamasfe.even-better-toml) extension. Can probably fix it to not do this if it matters.